### PR TITLE
[stable/opa] allow data loading configuration in kube-mgmt.

### PR DESCRIPTION
**Pull request by iverberk**
Signed-off-by: Ivo Verberk <iverberk@xebia.com>

@tsandall @timothyhinrichs @patrick-east 

#### Is this a new chart
NO

#### What this PR does / why we need it:

kube-mgmt supports data loading but the chart currently has no way of enabling that functionality, this PR fixes that. 

#### Which issue this PR fixes
N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by iverberk** [comment]
@timothyhinrichs @tsandall could anyone take a look and, if ok, merge? We really want to use the upstream chart instead of maintaining our own.

File: stable/opa/Chart.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ keywords:
 - opa
 - admission control
 - policy
-version: 1.13.1
+version: 1.13.2
```
Comment (timothyhinrichs) [inline stable/opa/Chart.yaml]: This conflicts with a just-merged PR that also bumped to 1.13.2

**Review by timothyhinrichs — COMMENTED** [review]
This change appears to be unnecessary.  The docs for kube-mgmt don't mention the `enable-data` flag; neither does the kube-mgmt:0.9 help, and the tutorial online doesn't use it.  https://www.openpolicyagent.org/docs/latest/kubernetes-tutorial/  

To replicate data, you just need to set the `--replicate` and `--replicate-cluster` flags to tell kube-mgmt what data to replicate, and that's already supported by the helm chart.  

Or am I missing the intent here?

**Comment by iverberk** [comment]
Hi Tim,

This change enables the data loading functionality that is described here: https://github.com/open-policy-agent/kube-mgmt/blob/master/README.md#json-loading

As far as I can tell this is unconfigurable right now. This PR also fixes a parameter in the chart that is not doing anything right now.

Regards,
Ivo

**Comment by dza89** [comment]
@timothyhinrichs 
https://github.com/open-policy-agent/kube-mgmt/blob/master/README.md#Json-loading

**Review by timothyhinrichs — COMMENTED** [review]
This looks good.  Sorry for not getting the intent earlier.  

Version bump needs to be 1.13.3.  Another PR merged underneath you.  Will approve once fixed.

**Comment by iverberk** [comment]
@timothyhinrichs version has been bumped 😀

**Pull request merged**